### ISSUE TITLE
ICE on Windows?

### ISSUE THREAD

**Issue by OmriSama** [peppermintos/ice#84]
Mozilla killed its SSB support, and now there isn't a way I know of to do isolated Firefox SSBs on Windows. I was wondering if it's possible to compile ICE on Windows, or if there's any other solutions that allow creating SSBs through Firefox.

Thanks!

**Comment by hemish04082005** [comment]
Well, it would be easy enough to make this program windows compatible, as gtk can compile and work on windows too.
But, the problem is that the program assumes a *nix (which includes linux and bsd) environment and has may hard coded things like "/usr/bin/firefox", so we need to rectify them to be cross platform, also it places ice.desktop file in /usr/share/applications which is kinda like a start menu entry. We need to edit path to windows compliant and instead of .desktop, we have to use .lnk for windows. Also, it places entries for web apps created with it in ~/.local/applications/ which is also hardcoded for *nix, we need to add windows directory too.
Basically, it is possible, and it is just 2-3 days of work, not much, i know python and read the code, it is very easy to do things in this small project, but sadly i don't know how to program gtk.
Well, if anyone else can, please do it as i also desparately need ssb's be it linux or windows. I could have stayed on firefox if ssb's were supported, i am now on chrome. (btw i use arch)

**Comment by OmriSama** [comment]
@Mark-Peppermint @PizzaLovingNerd any hints on this one?

**Comment by PizzaLovingNerd** [comment]
I would have to rewrite a lot of code for a platform that I don't even use, so ice will not support Windows unless someone else ports it over. But if you are on Linux, Ice still supports Firefox.

**Comment by grafiksinc** [comment]
As far I am concerned the code base will be adjusted  over the next few iterations of PeppermintOS anyway, @PizzaLovingNerd's fairly recent updates will be in Pep11, as time allows we are looking at using tkinter to replace gtk that solves your cross platform gui concerns,  @hemish04082005  is on point about the hard coding stuff. Also we gearing up to add a small database back-end for better, data management with the application .
